<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComDialog</class>
 <widget class="QDialog" name="ComDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>信道参数</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>串口号</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="ParityComboBox">
       <property name="currentText">
        <string>Even</string>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Odd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Even</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="PortNameComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="BaudRateComboBox">
       <property name="currentText">
        <string>2400</string>
       </property>
       <item>
        <property name="text">
         <string>600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>33600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>校验位</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>数据位</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="DataBitComboBox">
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>停止位</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="StopBitComboBox">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>260</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="ComButtonY">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ComButtonN">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="OpenComButton">
      <property name="text">
       <string>打开串口</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="CloseComButton">
      <property name="text">
       <string>关闭串口</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ComStatusLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>串口未开启！</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
